test/parallel/test-net-connect-immediate-destroy.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+var socket = net.connect(5000, '1.1.1.1', assert.fail);
```

Comment:
Can you use `common.PORT` and `common.localhostIPv4` for the port and host here.